qutip/visualization.py

```diff
@@ -7,7 +7,12 @@
            'matrix_histogram', 'plot_energy_levels', 'plot_fock_distribution',
            'plot_wigner', 'plot_expectation_values',
            'plot_spin_distribution', 'complex_array_to_rgb',
-           'plot_qubism', 'plot_schmidt']
+           'plot_qubism', 'plot_schmidt', '_cyclic_cmap', '_diverging_cmap',
+           '_sequential_cmap', '_is_fig_and_ax', '_cb_labels',
+           '_get_matrix_components', '_update_xaxis', '_update_yaxis',
+           '_update_zaxis', '_stick_to_planes', '_remove_margins',
+           '_set_ticklabels', '_blob', '_sequence_to_latex',
+           '_index_to_sequence', '_to_qubism_index_pair']
```

Comment:
Function that start with `_` are not mean to be made public and included here.
You can still import them using:
```
from .visualization import (_cyclic_cmap, _diverging_cmap, ...)
```